resources/process-openmaptiles.lua

```diff
@@ -826,7 +839,12 @@ end
 -- Set minimum zoom level by area but not below given minzoom
 function SetMinZoomByAreaWithLimit(minzoom)
 	local area=Area()
-	if     minzoom <= 6 and area>ZRES5^2  then MinZoom(6)
+	if     minzoom <= 1 and area>ZRES0^2  then MinZoom(1)
+	elseif minzoom <= 2 and area>ZRES1^2  then MinZoom(2)
+	elseif minzoom <= 3 and area>ZRES2^2  then MinZoom(3)
+	elseif minzoom <= 4 and area>ZRES3^2  then MinZoom(4)
+	elseif minzoom <= 5 and area>ZRES4^2  then MinZoom(5)
+	elseif minzoom <= 6 and area>ZRES5^2  then MinZoom(6)
```

Comment:
> 5. SetMinZoomByAreaWithLimit allows up to minzoom 4

I'm curious why does this only allow up to minzoom 4?
I'm trying to show lakes on all zoom levels. With this change they do now show up on zoom level 4 and 5, but not earlier.
E.g. https://www.openstreetmap.org/relation/945096

My config:
```json
"water":             { "minzoom": 0,  "maxzoom": 14, "simplify_below": 12, "simplify_level": 0.0003, "simplify_ratio": 2}
```

Reply:
Shouldn't the water now be displayed on all zoom levels if it is large enough when minzoom is set to 0?
For me it is not showing on zoom 0-3.